runtime/doc/term.txt

```diff
@@ -108,6 +108,11 @@ and right scroll margins as well.  If Nvim detects that the terminal is Xterm,
 it will make use of this ability to speed up scrolling that is not the full
 width of the terminal.
 
+							*tui-input*
+Nvim uses libtermkey to convert terminal escape sequences to key codes.
+|terminfo| is used first, and CSI sequences (including `CSI u` sequences) not
+in |terminfo| can also be parsed.
```

Comment:
the tmux example from your PR description would be useful here too. Concrete examples are super helpful to users.

Reply:
also worth mentioning:

> extended keys (also known as 'modifyOtherKeys' or CSI u)

Reply:
I just noticed one thing: making tmux always send a `CSI u` sequence can cause problems in programs that do not support `CSI u` sequence (e.g. cause pressing Escape to trigger undo in Vim), so such a example may actually cause problems.